### DISCUSSION TITLE
Change font and colors

### DISCUSSION THREAD

**Discussion by igorr0driguez** [hugo-themes/toha#719] [Q&A] [answered]
Hello,

Sorry if this is such a simple question, this is my first time ever using Hugo.  I followed everything in the "Quickstart" page and forked the sample repo.
I already started working on it and I want to manage the fonts used in the website and also customize the colors.
Although I searched everywhere for a CSS file or something, I can't seem to find it.

Thanks in advance!

**Comment by igorr0driguez** [comment, marked as answer]
Ok, I found the Customizing Layout page. Sorry for creating this discussion, feel free to delete it (I would have done it if I could).

**Reply by waylonli** [reply]
Could you please share how you change the font? I have the same issue...

**Comment by makidj88** [comment]
Hi everyone!
I deployed the site, configured it in regards of sections, bg images, logos, custom text, etc... but i really find it difficult to edit colors and fonts and such... for example i found the main .css file after building the static site public dir, but editing the .css file is pretty darn difficult since its minified. Ive tried re uploding the .css after converting it to non-minify or human readable, but the css breaks...  Ive scratched and searched for alternatives, different git repos were cloned, and now im confused as to how i edit the .css content. Ive tried creating the layouts directory, but no luck! For example, my fork/clone is different than one of the originals as it (the one i cloned/forked) does not have the whole "layout/*" structure pre-built...
Could someone please explain how i should go about this problem?
Which repo should i clone? How is the site configured beyond the sections and images...?
Thanks for any advice or help!
